app/Helpers/Helper.php

```diff
@@ -33,6 +33,16 @@ public static function parseEscapedMarkedown($str = null)
         }
     }
 
+    public static function parseEscapedMarkedownInline($str = null)
```

Comment:
Looks like we misspelled the original method as well but this should be `Markdown` and not `Markedown` 😅

Reply:
I don’t think that was a misspelling - Marked as in “this is marked down text”

Reply:
🤔  huh...my mind went to the text having been "markdowned" but that doesn't really make sense